### ISSUE TITLE
Hiding some words in commands during access in vscode

### ISSUE THREAD

**Issue by ahad-netlinks** [withfig/fig#2132]
### Checks

- [X] I have searched [github.com/withfig/fig/issues](https://github.com/withfig/fig/issues?q=) and there are no duplicates of my issue
- [X] I have run `fig doctor` in the affected terminal session
- [X] I have run `fig restart` and replicated the issue again

### Operating system

macOS 12.6.1 (21G217)

### Expected behaviour

When I write commands during access fig from vscode terminal some words don't appear in terminal

### Actual behaviour

didn't got one

### Environment

```yaml
fig-details:
  - 2.9.0
hardware-info:
  - model: 
  - model-id: 
  - chip-id: Intel(R) Core(TM) i7-6820HQ CPU @ 2.70GHz
  - cores: 4
  - mem: 16.00 GB
os-info:
  - macOS 12.6.1 (21G217)
environment:
  - shell: /bin/zsh
  - terminal: terminal
  - cwd: /Users/aahmadi
  - exe-path: /Users/aahmadi/.fig/bin/fig
  - install-method: unknown
  - env-vars:
    - FIGTERM_SESSION_ID: 774abe9e-b170-496a-9f2e-731164e662d7
    - FIG_HOSTNAME: ITAdmins-MacBook-Pro.local
    - FIG_SET_PARENT_CHECK: 1
    - FIG_TERM: 2.9.0
    - PATH: /usr/local/bin:/usr/bin:/bin:/usr/sbin:/sbin:/Users/aahmadi/.fig/bin:/Users/aahmadi/.local/bin
    - SHELL: /bin/zsh
    - TERM: xterm-256color
    - __CFBundleIdentifier: com.apple.Terminal
    - FIG_PID: 32288
    - FIG_SET_PARENT: 774abe9e-b170-496a-9f2e-731164e662d7
```

**Comment by grant0417** [comment]
It seems like VSCode is having some issues with text rendering, a screenshot or video here would be very useful.

**Comment by jwei98** [comment]
Having this same problem, I believe:

https://user-images.githubusercontent.com/19638700/207410536-e52d3902-896b-47bf-badc-8eda1d5efbca.mov

**Comment by grant0417** [comment]
@chaynabors is the script issues fixed in 2.10.0?

**Comment by grant0417** [comment]
@jwei98 that issue should be resolved now

@ahad-netlinks the issues appears to be upstream with vscode and being fixed soon https://github.com/microsoft/vscode/issues/166158
